build/validate.sh

```diff
@@ -69,7 +64,7 @@ fi
 
 echo -e "\nTesting that bosun starts and stops cleanly"
 #TODO: save linux build from above? cant seem to find where it is though.
-cd $GOPATH/src/bosun.org/cmd/bosun
+cd cmd/bosun
```

Comment:
Is there a reason to remove the full path from GOROOT rather than rely on relative pathing ?

Reply:
The only idea was to move out of the GOPATH since we're now using Go modules. Happy to add in an absolute path if you think that's clearer. 👍

Reply:
Tried this locally and got 
validate.sh: line 67: cd: cmd/bosun: No such file or directory
because I ran it from within the build directory
Probably worth putting the full path back in so that you can test the full linter setup locally without having to run from a specific directory